src/app/custom/translations.tsv

```diff
@@ -1037,6 +1037,7 @@
 "vega_variable_list"	"You can use these variables to set the size dynamically:"	"Vous pouvez utiliser ces variables pour définir la taille de manière dynamique :"
 "vega_variable_width"	"Replace with the width of the chart container"	"Remplacée par la largeur du conteneur du graphique"
 "vega_variable_height"	"Replace with the height of the chart container"	"Remplacée par la hauteur du conteneur du graphique"
+"vega_variable_container"	"Vega-Lite 'width' and 'height' variables to set the size of the graph to the same size as the parent container"	"Variables Vega-Lite pour 'width' et 'height' permettent de fixer la taille du graphique à la même taille que le conteneur parent"
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** singular‏
```suggestion
"vega_variable_container"	"Vega-Lite 'width' and 'height' variable to set the size of the graph to the same size as the parent container"	"Variable Vega-Lite pour 'width' et 'height' permettent de fixer la taille du graphique à la même taille que le conteneur parent"
```